kratos/containers/sparse_contiguous_row_graph.h

```diff
@@ -220,26 +221,38 @@ class SparseContiguousRowGraph
 
         if(rRowIndices.size() != nrows+1)
         {
-            rRowIndices.resize(nrows+1, false);
+            IndexType* pRowIndices = new IndexType[nrows+1];
+            rRowIndices = span<IndexType>(pRowIndices, nrows+1);
+            //rRowIndices.resize(nrows+1, false);
         }
-        //set it to zero in parallel to allow first touching
-        IndexPartition<IndexType>(nrows+1).for_each([&](IndexType i){
-            rRowIndices[i] = 0;
-        });
 
-        //count the entries 
-        IndexPartition<IndexType>(nrows).for_each([&](IndexType i){
-            rRowIndices[i+1] = mGraph[i].size();
-        });
-        
-        //sum entries
-        for(IndexType i = 1; i<static_cast<IndexType>(rRowIndices.size()); ++i){
-            rRowIndices[i] += rRowIndices[i-1];
+        if(nrows == 0) //empty
```

Comment:
Style is:

~~~c++
if (condition) {
}
~~~

Reply:
i formatted this with "astyle --style=allman" which is the one that i used since always... in any case, let's review this first in content, if we then agree about passing a different style formatter i will not resist